samples/subsys/display/lvgl/boards/waveshare_esp32s3_touch_lcd_1_28.conf

```diff
@@ -0,0 +1 @@
+CONFIG_MAIN_STACK_SIZE=4096
```

Comment:
Ohh, our minimal LVGL sample is not minimal anymore? What is the display resolution? Would you need same one-line-overlay for another LVGL sample?